Account/SyncAuthState.swift

```diff
@@ -104,28 +104,6 @@ extension SyncAuthStateCache: JSONLiteralConvertible {
     }
 }
 
-/// Moved from an old implementation of push crypto
-/// Used by the `token` function to get base64 data from the
-/// sync keys. Will be removed in https://github.com/mozilla-mobile/firefox-ios/issues/16005
-extension String {
-    /// Returns a base64 url safe decoding of the given string.
-    /// The string is allowed to be padded
-    /// What is padding?: http://stackoverflow.com/a/26632221
-    var base64urlSafeDecodedData: Data? {
-        // We call this method twice: once with the last two args as nil, 0 – this gets us the length
-        // of the decoded string.
-        let length = ece_base64url_decode(self, self.count, ECE_BASE64URL_REJECT_PADDING, nil, 0)
```

Comment:
This was the last relic of the OpenSSL library, which I have migrated to `Bytes.swift` extension (see below) and be more swift-oriented rather than OpenSSL